public/app/features/dashboard/components/PanelEditor/PanelEditor.tsx

```diff
@@ -55,14 +54,15 @@ import { notifyApp } from '../../../../core/actions';
 import { PanelEditorTableView } from './PanelEditorTableView';
 import { PanelModelWithLibraryPanel } from 'app/features/library-panels/types';
 import { getPanelStateForModel } from 'app/features/panel/state/selectors';
+import { getVariablesByKey } from '../../../variables/state/selectors';
```

Comment:
nit: would it be more clear to name this `getVariablesForDashboard` ?

Reply:
Good suggestion, but we can see a future where the key might not be related to a dashboard.
https://github.com/grafana/grafana/pull/44642#pullrequestreview-868768436

Reply:
hmm, could we find some other naming that makes this less generic? Feels like it, being this generic, adds even more complexity to an already complex concept. With that said, I don't have any good suggestions.

Reply:
`getVariablesInScope` ?

Reply:
And the scope would be the dashboard

Reply:
Maybe it should be `getVariablesByRootStateKey`

Reply:
hmm, naming is tricky 🙈  I don't think that names is more clear compared to the `getVariablesByKey` so I would say stick with the shorter one.